pkg/controller/installation/types.go

```diff
@@ -40,7 +40,7 @@ func newWorkshopType() *Type {
 	return &Type{
 		products: []v1alpha1.ProductName{v1alpha1.ProductAMQStreams},
 		productOrder: map[int][]v1alpha1.ProductName{
-			1: {v1alpha1.ProductAMQStreams},
+			1: {v1alpha1.ProductRHSSO, v1alpha1.ProductAMQStreams},
```

Comment:
I think RHSSO should be in phase 1, and AMQ Streams in phase 2. RHSSO should also be added to the managed install type.

Reply:
:+1:

Reply:
https://github.com/integr8ly/integreatly-operator/pull/18/commits/cc802cf0bce3f08ba61474a88af4c338a6c55228